app/routes/products_.$productId/route.tsx

```diff
@@ -74,6 +62,25 @@ export default function ProductDetailsPage() {
                         discountedPrice={product.priceData?.formatted?.discountedPrice}
                     />
                 )}
+
+                {product.productOptions?.map((option) =>
+                    option.name && option.choices ? (
+                        <ProductOption
```

Comment:
I think it makes sense to simply pass whole `option` instead of 5 props

Reply:
refactored